src/github/graphql.rs

```diff
@@ -0,0 +1,347 @@
+use anyhow::Context;
+use async_trait::async_trait;
+
+#[cynic::schema_for_derives(file = "src/github/github.graphql", module = "schema")]
```

Comment:
@jackh726 I'd add a bit of documentation [about the schema file](https://docs.github.com/en/graphql/overview/public-schema), especially about [updating it and maintenance](https://docs.github.com/en/graphql/overview/breaking-changes).

(I'm not familiar with GraphQL, so I don't know what maintenance implies in terms of context needed)